internal/cmd/stack/flags.go

```diff
@@ -30,3 +30,9 @@ var flagTail = &cli.BoolFlag{
 	Usage: "Indicate whether to tail the run",
 	Value: false,
 }
+
+var flagTailDefaultTrue = &cli.BoolFlag{
```

Comment:
One more thing actually - having a flag default to true feels a little strange for a CLI because you don't typically use them with an argument. For example you'd normally do:

```shell
spacectl some-command
```

or

```shell
spacectl some-command -tail
```

But you wouldn't usually do

```shell
spacectl some-command -tail false
```

Might just be worth having a think about it. Maybe flipping it and having an `-interactive` flag or something like that? That still has issues though because I get that you want to default to tailing in this case. 🤔 

Thinking about this now - are we using the right term here at all? I'd normally think of `tail` as "return the last X records", and `follow` as "keep returning results as they're available".

Reply:
Maybe I'll just add -notail ?

Reply:
Yeah - that could work. It means there's not two separate defaults for `-tail` depending on the command used as well.

Reply:
Done.